hathor/cli/run_node.py

```diff
@@ -296,6 +298,13 @@ def create_wallet():
         if args.x_fast_init_beta:
             self.log.warn('--x-fast-init-beta is now the default, no need to specify it')
 
+        if args.enable_event_queue:
+            self.manager.enable_event_queue = True
+            self.log.info('--enable-event-queue flag provided." \
+                           The events detected by the full node will be stored and retrieved to clients')
+
+            self.manager.retain_events = args.retain_events is True
```

Comment:
There is one case, where `--retain-events` is passed, but `--enable-event-queue` is not, that gives an outcome where `retain_events == False` (thanks for the table, it helped spot this easily).

I think that behavior isn't good. I think the better behavior would be to refuse to start, since it's very likely that the use forgot about `--enable-event-queue` in that case.

Reply:
I see this was addressed bellow.